packages/core/admin/webpack.config.dev.js

```diff
@@ -2,10 +2,13 @@
 
 const path = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const { DuplicateReporterPlugin } = require('duplicate-dependencies-webpack-plugin');
+
 const webpackConfig = require('./webpack.config.js');
 
 module.exports = () => {
   const analyzeBundle = process.env.ANALYZE_BUNDLE;
+  const analyzeDuplicateDependencies = process.env.ANALYZE_DEPS;
```

Comment:
In which case ANALYZE_DEPS will be set to false?

Reply:
It will be set to undefined if it does not exist. But I made sure to check if the env variable is defined that we analyse the bundle when it is explicitly set to `true`